docs/team/chuckiex3.md

```diff
@@ -0,0 +1,23 @@
+# Mooi Suet Yeng, Francene's Project Portfolio Page
+
+## Project: Zer0Note
+//todo standardise
+
+### Summary of Contributions
+- Created ASCII art for Zer0Note
+- New Feature: Added the ability to add tasks and notebooks/sections/pages
+- New Feature: Added the ability to select a notebook/section/page
```

Comment:
A bit too minimal. Maybe you could elaborate more on the Justifications and Highlights?

Reply:
heheheh.... I will craft something... thanks xD